Cargo.toml

```diff
@@ -25,15 +25,16 @@ http-body = "0.4.5"
 hiro-system-kit = {version = "0.1.0", features = ["log"]}
 strum_macros = "0.24.3"
 strum = "0.24.1"
+time = "0.3.35"
```

Comment:
Ok I see were this come from, the `time` version we had doesn't compile with the latest rust version.

I don't think we should add time as an explicit dependency since the project doesn't directly depend on it. It's a sub-dependency.

Running `$ cargo update time` will update the cargo.lock to the latest compatible version (and fix the issue).

It seems that it's doen already for this PR, so we can just remove this line.
Sorry for not catching it in my first review